### PR TITLE
Fix for the password length error message during setup 

### PR DESCRIPTION
**Pull request by JoshuaLuckers**
### What does it do?
Changes the error message you get when the password is to short (when creating an account during setup)

### Why is it needed?
To fix issue #14109

### Related issue(s)/PR(s)
#14109

### CODE REVIEW

**Comment by JoshuaLuckers** [comment]
Do I have to change the lexicons in other languages as well or is this done via Crowdin?

**Comment by Jako** [comment]
That has be done on Crowdin.

File: setup/lang/en/default.inc.php [hunk 1]
```diff
@@ -186,7 +186,7 @@
 $_lang['password_err_invchars'] = 'Your password may not contain any invalid characters, such as /, \\, &apos;, &quot;, (, ) or {}.';
 $_lang['password_err_nomatch'] = 'Does not match password';
 $_lang['password_err_ns'] = 'Password is empty';
-$_lang['password_err_short'] = 'Your password must be at least 6 characters long.';
+$_lang['password_err_short'] = 'Your password must be at least 8 characters long.';
```
Comment (Jako) [inline setup/lang/en/default.inc.php:189]: Maybe the 8 could be replaced by a placeholder [[+length]] and set that value in the lexicon call.

**Review by Jako — CHANGES_REQUESTED** [review]

**Comment by Jako** [comment]
Closing this in favour of #14112

**Pull request closed without merging**